response_operations_ui/views/admin/manage_user_accounts.py

```diff
@@ -281,6 +282,16 @@ def post_manage_account_groups(user_id):
     return redirect(url_for("admin_bp.manage_user_accounts"))
 
 
+def delete_user_from_redis_session(user_id):
+    all_sessions = current_app.config["SESSION_REDIS"]
+    get_all_sessions_keys = all_sessions.scan(cursor=0, match="*")[1]
+
+    for user_in_session in get_all_sessions_keys:
+        user_session_information = str(all_sessions.mget(user_in_session))
+        if user_id in user_session_information:
+            all_sessions.delete(user_in_session)
```

Comment:
Can you put a comment in this loop to say why we're not exiting as soon as we find the user?  I didn't realise you could have multiple sessions at once and that's a fact that's easily forgotten by the team!